### DISCUSSION TITLE
Does Kong 3.4 LTS support Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by hexclann** [Kong/kong#13897] [Help] [answered]
Hello, does Kong support Ubuntu 24.04? I couldn't see anything in the documentation regarding the 24.04 support. There are few GitHub actions which runs on 24.04 apart from this, I couldn't find any info

**Comment by fffonion** [comment, marked as answer]
It doesn't. The next LTS version will include Ubuntu 24.04.
